src/withReducer.js

```diff
@@ -72,8 +75,10 @@ const withReducer = (
     render() {
       return React.createElement(WrappedComponent, {
         ...this.props,
-        ...this.state,
-        ...this.actionCreators
+        ...(this.mapToProps({
+          ...this.state,
```

Comment:
We should pass state and action creators to `mapToProps`, in case someone wants to namespace them separately.

```js
this.mapToProps(this.state, this.actionCreators)
```